### PR TITLE
Properly handle GitHub repos

### PR DESCRIPTION
**Pull request by szepeviktor**
Using raw URL maybe ratelimited etc.

### CODE REVIEW

File: .github/workflows/update-build-files.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Update WordPress i18n build files
 
 on:
   schedule:
-    - cron: "0 0 * * *"  # Runs every day at midnight.
+    - cron: "0 0 * * *"  # Funciona todos os dias à meia-noite.
```
Comment (szepeviktor) [inline .github/workflows/update-build-files.yml:5]: # 🇵🇹

File: .github/workflows/update-build-files.yml [hunk 2]
```diff
@@ -33,22 +33,25 @@ jobs:
...
 
     - name: Check if source files have changed
       id: compare-files
       run: |
         if ! cmp -s downloads/readme.html en_US/readme.html || ! cmp -s downloads/wp-config-sample.php en_US/wp-config-sample.php; then
           echo "The build files have been updated!"
           echo "files_changed=true" >> $GITHUB_OUTPUT
-        else
-          echo "The build files remain unchanged."
-          echo "files_changed=false" >> $GITHUB_OUTPUT
```
Comment (szepeviktor) [inline .github/workflows/update-build-files.yml:51]: This is unused code.

Comment (pedro-mendonca) [reply .github/workflows/update-build-files.yml:51]: This code is used.
https://github.com/wp-portugal/wp-portugal-i18n-build-files/actions/runs/8714352677/job/23904450109

imagem

Comment (szepeviktor) [reply .github/workflows/update-build-files.yml:51]: I mean the output variable is never checked for "false", only for "true".

**Comment by zedejose** [comment]
Why, thank you so so much @szepeviktor, (we never imagined that our tiny little repo would catch anyone's eye, much less phpstan's :) ). We'll be just tweaking the translation after the merge, but otherwise it looks much better now

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**